.github/workflows/python-publish.yml

```
@@ -0,0 +1,62 @@
+        rm setup.py.bak # Remove backup file
+        echo "setup.py content after update:"
+        cat setup.py
+
+    - name: Build Python package
+      working-directory: python
+      run: |
+        python -m pip install --upgrade pip
+        pip install build twine
+        python -m build
+    - name: Publish to PyPI
+      working-directory: python
```

actionlint (1.7.4)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)